website-1.x/blog/2021-05-11-releasing-docusaurus-beta.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Releasing Docusaurus 2 beta
+author: Sébastien Lorber
+authorTitle: Docusaurus maintainer
+authorURL: https://sebastienlorber.com
+authorImageURL: https://github.com/slorber.png
+authorTwitter: sebastienlorber
+tags: [release, beta]
+# image: /img/blog/2021-03-09-releasing-docusaurus-i18n/social-card.png
+---
+
+Today, we officially release the first **Docusaurus 2 beta**, after a **very long alpha period**.
+
+We believe Docusaurus 2 is ready for **mainstream adoption**!
+
+<!--truncate-->
+
+## Docusaurus adoption
+
+**Don't be afraid of the beta tag!**
+
+Docusaurus 2 is **widely adopted** and growing fast:
+
+![Docusaurus growth](/img/blog/2021-05-11-releasing-docusaurus-beta/trend.png)
+
+Our new [showcase](https://docusaurus.io/showcase) page allows you to **filter Docusaurus sites by features**: get inspired by real-world production sites with a similar use-case!
+
+Don't miss our [favorite](https://docusaurus.io/showcase?tags=favorite) sites, they all **stand out** with something unique:
+
+[![Docusaurus growth](/img/blog/2021-05-11-releasing-docusaurus-beta/favorites.png)](https://docusaurus.io/showcase?tags=favorite)
+
+## Why was Docusaurus v2 in alpha for so long?
+
+The first alpha release [v2.0.0-alpha.0](https://github.com/facebook/docusaurus/releases/tag/v2.0.0-alpha.0) was **2 years ago** 😅.
+
+It was definitively a very long and unusual alpha period.
+
+Docusaurus 2 has reached **feature parity** with Docusaurus 1 with **[i18n](https://docusaurus.io/blog/2021/03/09/releasing-docusaurus-i18n)**, and it has been **successfully adopted** by many Docusaurus sites (see [i18n showcase](https://docusaurus.io/showcase?tags=i18n)).
```

Comment:
```suggestion
As this was a substantial re-architecture of the code base, we wanted to ensure that we had confidence in the stability and features of Docusaurus 2 before moving on to a beta stage, since beta suggests a high level of quality. We are happy to say that Docusaurus 2 has reached **feature parity** with Docusaurus 1 with **[i18n](https://docusaurus.io/blog/2021/03/09/releasing-docusaurus-i18n)**, and it has been **successfully adopted** by many Docusaurus sites (see [i18n showcase](https://docusaurus.io/showcase?tags=i18n)).
```